.travis.yml

```diff
@@ -13,6 +13,7 @@ before_install:
   - cd libsodium && sudo ./configure && sudo make check && sudo make install && cd ..
 
 cache:
+  ccache: true
```

Comment:
for what do you need it? is this to speed the build of rocksdb? there is a branch that speed it. Also eventually ccache can be added to it. Let me know :)